myParser.py

```diff
@@ -79,15 +91,13 @@ def parse():
         "-m",
```

Comment:
Can you edit the Usage section of the README to reflect the update use of this command?

Reply:
Will do